src/Pos/WorkMode.hs

```diff
@@ -149,8 +108,8 @@ type RawRealMode ssc =
     SscHolder ssc (
     NtpSlotting (
     SlottingHolder (
-    ContextHolder ssc (
     DBHolder (
+    ContextHolder ssc (
```

Comment:
It's still not clear why `ContextHolder ssc (DBHolder _)` is worse than `DBHolder (ContextHolder ssc _)`. I can't say it's better, I just don't understand whether there is any significant difference.